localstack-core/localstack/services/ssm/resource_providers/aws_ssm_parameter.py

```diff
@@ -203,6 +223,7 @@ def list(
         return ProgressEvent(
             status=OperationStatus.SUCCESS,
             resource_models=[
-                SSMParameterProperties(Id=resource["Name"]) for resource in resources["Parameters"]
+                SSMParameterProperties(Name=resource["Name"])
+                for resource in resources["Parameters"]
```

Comment:
@Pive01 Will this create an issue with the resource overview? If so please let me know so that I don't break it for you! :wink:

Reply:
Actually, I was just watching your demo and realised that the ssm parameter didn't. So I realised that they are actually broken and this changes fixes it. We should probably try to push to merge this before the patch release!

Reply:
Thanks @cloutierMat , it does indeed fix an issue as the identifier was not set correctly...now is properly discovered :rocket: